source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -1,871 +1,865 @@
 
-#. Create a symbolic link.
+* Docker proxy (optional)
 
-   .. code-block:: bash
+  If you use a proxy server, you must set your proxy environment variables and
+  create a proxy configuration file for the Docker daemon and container.
 
-      sudo ln -sf /home/clr/mix/update/www /var/www/mixer
+  Consult your IT department if you are behind a corporate proxy for the correct
+  values.
 
-   .. note::
+  Refer to `Configure Docker proxy info`_ for instruction.
 
-      `/home/clr/mix` is the workspace for the mix (read `Create workspace`)
+* Location to host the update content and images
 
+  In order for swupd to make use of your mix, the update content for your mix
+  must be hosted on a web server. Your mix will be configured with an update
+  location URL, which swupd will use to pull down updates.
 
-#. Set up ``nginx`` configuration. 
+  Refer to `Set up a nginx web server for mixer`_ for an simple example of
+  setting up an update location.
 
-   .. code-block:: bash
+First time setup
+================
 
-      sudo mkdir -p  /etc/nginx/conf.d
+First time setup creates and initializes the workspace used by mixer. Complete
+setup before you create a mix.
 
-#. Copy the default example configuration file. 
+#. Create workspace.
 
-   .. code-block:: bash
+   The mixer tool uses a simple workspace to contain all input and output in a
+   basic directory structure. The workspace is simply an empty folder that you
+   will execute the mixer commands from. Each mix will use its own separate
+   workspace.
 
-      sudo cp -f /usr/share/nginx/conf/nginx.conf.example /etc/nginx/nginx.conf
+#. Initialize the workspace and mix.
 
-#. Open ``mixer.conf`` with an editor.
+   Before you create a mix, you must explicitley initialize the mixer workspace.
+   During initialization, the mixer workspace is configured and the base for
+   your mix is defined. By default, your mix will be based on the latest
+   upstream version and start with the minimum set of bundles. Your first custom
+   mix version number will start at 10. You can alternately select other
+   versions or bundle sets to start from.
 
-   .. code-block:: bash
- 
-      sudo nano /etc/nginx/conf.d/mixer.conf 
+   Initialization creates the directory structure used by mixer and adds the
+   :file:`builder.conf` file, which is used to configure the mixer tool.
 
-#. Add the server configuration content below to ``mixer.conf``. 
+   View the `mixer.init man page`_ for more information on mixer
+   initialization.
 
-   .. code-block:: console
+   View the list of `suitable versions`_ to mix from.
 
-      server {
-           server_name localhost;
-           location / {
-                     root /var/www/mixer;
-                     autoindex on; 
-           }
-      }
+#. Edit builder.conf.
 
-#. Restart the daemon, enable nginx on boot, and start the service.
+   :file:`builder.conf` is used to configure the mixer tool. For example, it
```

Comment:
I wouldn't say that "builder.conf" is used to configure the mixer tool.  We're not configuring the mixer tool per se. 
 Instead, mixer uses configure information stored in builder.conf to build the mix.  For example, etc...